addons/mail/models/mail_alias.py

```diff
@@ -228,22 +255,43 @@ def _get_alias_bounced_body_fallback(self, message_dict):
         contact_description = self._get_alias_contact_description()
         default_email = self.env.company.partner_id.email_formatted if self.env.company.partner_id.email else self.env.company.name
         return Markup(
-            _("""<p>Dear Sender,<br /><br />
-The message below could not be accepted by the address %(alias_display_name)s.
+            '<p>' +
+            _('Dear Sender') + ',<br /><br />\n' +
+            _("""The message below could not be accepted by the address %(alias_display_name)s.
 Only %(contact_description)s are allowed to contact it.<br /><br />
-Please make sure you are using the correct address or contact us at %(default_email)s instead.<br /><br />
-Kind Regards,</p>"""
-             )) % {
-                 'alias_display_name': self.display_name,
-                 'contact_description': contact_description,
-                 'default_email': default_email,
-             }
+Please make sure you are using the correct address or contact us at %(default_email)s instead.""") + '<br /><br />' +
+            _('Kind Regards,') +
+            '</p>') % {
+                   'alias_display_name': self.display_name,
+                   'contact_description': contact_description,
+                   'default_email': default_email,
+               }
 
     def _get_alias_contact_description(self):
         if self.alias_contact == 'partners':
             return _('addresses linked to registered partners')
         return _('some specific addresses')
 
+    def _get_invalid_alias_bounced_body(self, message_dict):
```

Comment:
If we have "_get_alias_bounced_body_fallback", maybe this one could be "_get_alias_invalid_body" ?

Reply:
Yes that’s a good name, thanks.